canonical-web-and-design/canonical.com
# Fix broken products menu

## Done

Remove underscore that precedes the products navigation partial so it can be rendered in the browser. Also did the same for the careers navigation partial even though it currently isn't used.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/start
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click the "Products" navigation and check that the products mega menu is displayed


## Issue / Card

Fixes #228
